diesel_cli/src/print_schema.rs

```diff
@@ -100,7 +100,14 @@ impl<'a> Display for ColumnDefinitions<'a> {
             let mut out = PadAdapter::new(f);
             writeln!(out, "{{")?;
             for column in self.0 {
-                write!(out, "{} -> ", column.name)?;
+                if let Some(ref new_name) = column.new_name {
+                    write!(out, "#[sql_name = \"{}\"]\n", column.name)?;
```

Comment:
Would need to use `writeln!` as well since `\n` doesn't work with raw strings (which this should do regardless)